resources/i18n/pt_BR/cura.po

```diff
@@ -541,12 +541,12 @@ msgstr "Houve um erro ao conectar à nuvem."
 #: /home/ruben/Projects/Cura/plugins/UM3NetworkPrinting/src/Cloud/CloudProgressMessage.py:14
 msgctxt "@info:status"
 msgid "Sending Print Job"
-msgstr ""
+msgstr "Enviando Trabalho de Impressão"
 
 #: /home/ruben/Projects/Cura/plugins/UM3NetworkPrinting/src/Cloud/CloudProgressMessage.py:15
 msgctxt "@info:status"
 msgid "Uploading via Ultimaker Cloud"
-msgstr ""
+msgstr "Transferindo via Nuvem Ultimaker"
```

Comment:
You don't have to translate "Ultimaker Cloud", it's the name of the product or feature.